examples/langchain-langgraph/test_langchain_mcp.py

```
@@ -0,0 +1,65 @@
+                "url": "http://localhost:8080/mcp",
+                "transport": "streamable_http",
+            }
+        }
+
+    )
+    mcp_tools = await mcp_client.get_tools()
+    # reduce the tool list to just 'namespaces_list'
+    mcp_tools = [tool for tool in mcp_tools if tool.name == 'namespaces_list']
+    llm_with_mcp_tools = llm.bind_tools(mcp_tools)
+    # just proving mcp_tools can be passed into the 'ToolNode' constructor
+    mcp_tool_node = ToolNode(mcp_tools)
```

Ruff (0.14.2)

Local variable `mcp_tool_node` is assigned to but never used

Remove assignment to unused variable `mcp_tool_node`

(F841)